deployment/ansible/roles/model-my-watershed.celery-worker/templates/celery-syslog.j2

```diff
@@ -0,0 +1,8 @@
+module(load="imfile")
+
+{% for id in range(0,celery_number_of_workers) %}
+input(type="imfile"
+      File="/var/log/celery/*-worker{{ id }}.log"
+      stateFile="imfile-state:-var-log-celery-worker{{ id }}.log"
```

Comment:
It's worth surfacing that rsyslog doesn't like that we're defining `stateFile` while we're using a wildcard in `File`:

```
Jul 01 12:22:51 ip-10-0-5-105 rsyslogd-2207:  error during parsing file /etc/rsyslog.d/23-celery.conf, on or before line 6: parameter 'statefile' deprecated but accepted, consider removing or replacing it [v8.16.0 try http://www.rsyslog.com/e/2207 ]
Jul 01 12:22:51 ip-10-0-5-105 rsyslogd-2207:  error during parsing file /etc/rsyslog.d/23-celery.conf, on or before line 10: parameter 'statefile' deprecated but accepted, consider removing or replacing it [v8.16.0 try http://www.rsyslog.com/e/2207 ]
Jul 01 12:22:51 ip-10-0-5-105 rsyslogd:  rsyslogd's groupid changed to 108
Jul 01 12:22:51 ip-10-0-5-105 rsyslogd:  rsyslogd's userid changed to 104
Jul 01 12:22:51 ip-10-0-5-105 rsyslogd-2420:  imfile: warning: it looks like to-be-monitored file "/var/log/celery/*-worker0.log" contains wildcards. This usually does not work well with specifying a state file. [v8.16.0 try http://www.rsyslog.com/e/2420 ]
Jul 01 12:22:51 ip-10-0-5-105 rsyslogd-2420:  imfile: warning: it looks like to-be-monitored file "/var/log/celery/*-worker1.log" contains wildcards. This usually does not work well with specifying a state file. [v8.16.0 try http://www.rsyslog.com/e/2420 ]
...
Jul 01 12:23:13 ip-10-0-5-105 rsyslogd-2361:  imfile: state file '/var/spool/rsyslog/imfile-state:-var-log-celery-worker0.log' contains file name '/var/log/celery/Black-worker0.log', but is used for file '/var/log/celery/Blue-worker0.log'. State file deleted, starting from begin of file. [v8.16.0 try http://www.rsyslog.com/e/2361 ]
Jul 01 12:23:15 ip-10-0-5-105 rsyslogd-2361:  imfile: state file '/var/spool/rsyslog/imfile-state:-var-log-celery-worker1.log' contains file name '/var/log/celery/Black-worker1.log', but is used for file '/var/log/celery/Blue-worker1.log'. State file deleted, starting from begin of file. [v8.16.0 try http://www.rsyslog.com/e/2361 ]
```

This came up towards the end of the effort to get Xenial working on staging. I did some research, and the rsyslog documentation is a bit heavy. It may be worth splitting this out.

Reply:
I think that we can just get rid of the deprecated `stateFile` parameter here. That appears to be suppressing the warnings, and logs are still flowing.